bower.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "purescript-leafletjs-halogen",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "output"
+  ],
+  "dependencies": {
+    "purescript-eff": "^3.1.0",
+    "purescript-halogen": "^2.0.1",
+    "purescript-leafletjs": "^0.2.0",
+    "purescript-halogen-css": "^6.0.0"
+  },
+  "devDependencies": {
+    "purescript-debug": "^3.0.0",
+    "purescript-random": "^3.0.0"
+  }
+}
```

Comment:
This is missing a few fields if we want to be able to publish it to pursuit - `license` and `repository` I think?